docs/redisjson.md

```diff
@@ -0,0 +1,98 @@
+# RedisJSON Jedis Quick Start
+
+Jedis supports [RedisJSON](https://oss.redis.com/redisjson/) and [RediSearch](https://oss.redis.com/redisearch/).
```

Comment:
If only `RedisJSON` and `RediSearch` are mentioned here, then add another Module and need to modify this, so I suggest deleting this line. In the `RedisJSON` documentation, only the usage of `RedisJSON` is provided.

Reply:
@yangbodong22011 I don't mind this. At some point in history, we did 

https://github.com/redis/jedis/blob/f4b8df496c698b17dc8a009b704de39de8bcfdc7/README.md#L24-L46

So we can afford listing module names that supported through Jedis.

Reply:
@yangbodong22011 I've re-read and I believe we misunderstood. This doc describes RedisJSON with RediSearch. From that perspective, there's no problem with the statement.

Reply:
@sazzad16 yes, agree with you.